source/reference/config/triggers.txt

```diff
@@ -32,9 +39,13 @@ trigger type. The following fields exist in *all* trigger configuration files:
    {
```

Comment:
IDK if it matters or if we want to update this as part of the other ticket, but my config files are generating fields in alphabetical order

Reply:
I'll update as part of other ticket!